woocommerce/woocommerce-gateway-stripe
# Add webhook information to api key dialog

## Changes proposed in this Pull Request:







Adds more informative text about the webhook secret and what merchants must do to enable webhooks on their site, as per [new designs](https://www.figma.com/file/Igkr3nOx0UoGDlHrsJ4eux/%5BStripe%5D-Universal-Payment-Element-(UPE)?node-id=959%3A59586).

- Add a instructions to the API key dialog.
- Update the text in the account setup dialog.


## Testing instructions





1. Set up a fresh WooCommerce site.
2. During account setup choose to enter your API keys manually and make sure there's text that informs you of what the webhook endpoint is and where to update your Stripe settings.
3. Go to the Stripe plugin's settings.
4. Open the **Settings** tab and click **Edit account keys**.
5. Make sure there's text that informs you of what the webhook endpoint is and where to update your Stripe settings.

<img width="571" alt="image" src="https://user-images.githubusercontent.com/13835680/144392144-79f13a45-0f23-478d-bb77-6a0a6d092b48.png">

---


**Post merge**
